macos/generate.sh

```diff
@@ -12,10 +12,10 @@ file=binutils-$arch.rb
 
 cat > $file <<EOF
 class Binutils$(title_case $arch) < Formula
-  homepage "http://www.gnu.org/software/binutils/binutils.html"
-  url "http://ftpmirror.gnu.org/binutils/binutils-2.25.tar.gz"
-  mirror "http://ftp.gnu.org/gnu/binutils/binutils-2.25.tar.gz"
-  sha1 "f10c64e92d9c72ee428df3feaf349c4ecb2493bd"
+  homepage "https://www.gnu.org/software/binutils/binutils.html"
+  url "https://ftpmirror.gnu.org/binutils/binutils-2.34.tar.gz"
+  mirror "https://ftp.gnu.org/gnu/binutils/binutils-2.34.tar.gz"
+  sha256 "53537d334820be13eeb8acb326d01c7c81418772d626715c7ae927a7d401cab3"
```

Comment:
Checksum verification seems enough to me, why use https for download? (I approve https homepage, though)

Reply:
I just did a search replace for all http.  It doesn't even look like I can use this on mac due to the qemu.  So.. however you want to do it - I was just trying to keep it up to date.

Reply:
Alright, whatever, I am too busy lately to decide stuff, so I'll just go with this.